### PR TITLE
Updates Soy attributes declaration

### PR DESCRIPTION
**Pull request by fernandosouza**
cc @brunobasto

### CODE REVIEW

File: src/pages/docs/getting-started/modal.md [hunk 1]
```diff
@@ -30,10 +30,11 @@ modal. Let's update the component's template to do that then:
 /**
  * This renders the component's whole content.
  * Note: has to be called ".render".
- * @param body
- * @param header
  */
 &#123;template .render&#125;
+    &#123;@param body: string&#125;
+    &#123;@param header: string&#125;
+
```
Comment (blzaugg) [inline src/pages/docs/getting-started/modal.md]: I don't think `&#123;` and `&#125;` is necessary with `@param`. I think we only need it for `template` and `namespace`.
 - [x] Can you try it with just `{` and `}`?

Comment (fernandosouza) [reply src/pages/docs/getting-started/modal.md]: Done, @blzaugg. Thx. 😃

File: src/pages/docs/getting-started/modal.md [hunk 2]
```diff
@@ -30,10 +30,11 @@ modal. Let's update the component's template to do that then:
 /**
  * This renders the component's whole content.
  * Note: has to be called ".render".
- * @param body
- * @param header
  */
 &#123;template .render&#125;
+    {@param body: string};
+    {@param header: string};
+
```
Comment (blzaugg) [inline src/pages/docs/getting-started/modal.md]: - [ ] Remove training `;`, throughout files.

**Comment by blzaugg** [comment]
Besides the trailing `;`, LGTM.

**Comment by fernandosouza** [comment]
@blzaugg. Fixed.

**Pull request merged**